Src/CSharpier/CodeFormatter.cs

```diff
@@ -14,12 +14,33 @@ namespace CSharpier
 {
     public class CodeFormatter
     {
-        public CSharpierResult Format(string code, PrinterOptions printerOptions)
+        public static string Format(string code, CodeFormatterOptions? options = null)
```

Comment:
Why is this necessary? With C#9 having async main, there's really no reason to do `.Result` anymore.

Reply:
If you mean why have both `Format` and `FormatAsync` it seems like a nuget library should supply sync and async versions of a method because there are cases where someone may want to format code but they aren't in an async method. From what I know source generators aren't async.

Reply:
That is what I meant. I figured if we're just calling `.Result` anyway, we may as well leave that up to the caller to deal with.
That being said, I don't feel too strongly about it so If you think it's a worthwhile addition, let's go ahead with it.